packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -0,0 +1,469 @@
+import { useViewportMatch } from '@wordpress/compose';
+import { createInterpolateElement } from '@wordpress/element';
+import { sprintf } from '@wordpress/i18n';
+import { useI18n } from '@wordpress/react-i18n';
+import classnames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import { noop } from 'lodash';
+import { useMemo } from 'react';
+import {
+	DEFAULT_VIEWPORT_WIDTH,
+	DEFAULT_VIEWPORT_HEIGHT,
+	MOBILE_VIEWPORT_WIDTH,
+} from '../constants';
+import {
+	getDesignPreviewUrl,
+	getMShotOptions,
+	isBlankCanvasDesign,
+	filterDesignsByCategory,
+	sortDesigns,
+} from '../utils';
+import BadgeContainer from './badge-container';
+import { UnifiedDesignPickerCategoryFilter } from './design-picker-category-filter/unified-design-picker-category-filter';
+import ThemePreview from './theme-preview';
+import type { Categorization } from '../hooks/use-categorization';
+import type { Design } from '../types';
+import './style.scss';
+
+const makeOptionId = ( { slug }: Design ): string => `design-picker__option-name__${ slug }`;
+
+interface DesignPreviewImageProps {
+	design: Design;
+	locale: string;
+	highRes: boolean;
+}
+
+const DesignPreviewImage: React.FC< DesignPreviewImageProps > = ( { design, locale, highRes } ) => {
+	const scrollable = design.preview !== 'static';
+	const isMobile = useViewportMatch( 'small', '<' );
+
+	return (
+		<MShotsImage
+			url={ getDesignPreviewUrl( design, { language: locale, use_screenshot_overrides: true } ) }
+			aria-labelledby={ makeOptionId( design ) }
+			alt=""
+			options={ getMShotOptions( { scrollable, highRes, isMobile } ) }
+			scrollable={ scrollable }
+		/>
+	);
+};
+
+interface DesignButtonProps {
+	design: Design;
+	locale: string;
+	onSelect: ( design: Design ) => void;
+	premiumBadge?: React.ReactNode;
+	highRes: boolean;
+	disabled?: boolean;
+	hideFullScreenPreview?: boolean;
+	hideDesignTitle?: boolean;
+	hasDesignOptionHeader?: boolean;
+	isPremiumThemeAvailable?: boolean;
+	onCheckout?: any;
+}
+
+const DesignButton: React.FC< DesignButtonProps > = ( {
+	locale,
+	onSelect,
+	design,
+	premiumBadge = null,
+	highRes,
+	disabled,
+	hideDesignTitle,
+	hasDesignOptionHeader = true,
+	isPremiumThemeAvailable = false,
+	onCheckout = undefined,
+} ) => {
+	const { __ } = useI18n();
+
+	const badgeType = design.is_premium ? 'premium' : 'none';
+
+	const badgeContainer = ! isEnabled( 'signup/theme-preview-screen' ) ? (
+		design.is_premium && premiumBadge
+	) : (
+		<BadgeContainer badgeType={ badgeType } isPremiumThemeAvailable={ isPremiumThemeAvailable } />
+	);
+
+	const shouldUpgrade = design.is_premium && ! isPremiumThemeAvailable;
+
+	function getPricingDescription() {
+		if ( ! isEnabled( 'signup/theme-preview-screen' ) ) {
+			return null;
+		}
+
+		let text: any = __( 'Free' );
+
+		if ( design.is_premium ) {
+			text = createInterpolateElement(
+				shouldUpgrade
+					? __( '<button>Included in the Pro plan</button>' )
```

Comment:
We need to change this copy, see: https://github.com/Automattic/wp-calypso/pull/65843

Reply:
good catch
1